spec/api/comment_thread_spec.rb

```diff
@@ -477,6 +477,18 @@ def test_unicode_data(text)
         check_thread_result_json(nil, thread, parsed)
       end
 
+      context 'when requesting the thread for informational purposes' do
+        subject do
+          get "/api/v1/threads/#{thread.id}", with_responses: false # we're asking for no responses here.
```

Comment:
I'm fine with your proposal. Also, let me know if you plan on doing the work to reduce the number of calls from lms on create response as well.

Thanks again.